assert/array_includes.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
 // This module is browser compatible.
 import { equal } from "./equal.ts";
-import { format } from "@std/internal/format";
+import { format } from "../internal/format.ts";
```

Comment:
This relative import doesn't work when published to JSR as each package is published independently

Reply:
I made that change because the `@std` import led to this error when using the repo with `file:///` imports from 3rd party code:

`Relative import path "@std/internal/build-message" not prefixed with / or ./ or ../ and not in import map`

Is there a syntax that would work in both cases, or should I revert the change from the first commit of this PR?